### PR TITLE
Show value store QRs if available

### PR DESCRIPTION
**Pull request by carelvwyk**

### CODE REVIEW

**Comment by ckritzinger** [comment]
The binance QR looks nice but for me it didn't work in all circumstances. I might be wrong but you should probably test all of the following:

- scan QR with the device camera
- scan QR with Binance app

**Review by ckritzinger — APPROVED** [review]

File: src/components/WalletSelect.vue [hunk 1]
```diff
@@ -37,6 +37,11 @@ export default {
                       <img src="@/assets/wallets/lightning.png" class="object-contain w-full h-full"></img>
                    </button>
                 </li>
+                <li>
```
Comment (ckritzinger) [inline src/components/WalletSelect.vue:40]: I think we should think carefully about the optics of having VALR introduce us to a merchant and then having their logo be right at the bottom of the payment options.

**Comment by carelvwyk** [comment]
> The binance QR looks nice but for me it didn't work in all circumstances. I might be wrong but you should probably test all of the following:
> 
> * scan QR with the device camera
> * scan QR with Binance app

TEST RESULTS
* Scannable with device camera - launches Binance app and payment details
* Scannable with Binance app - launches payment details

File: src/components/WalletSelect.vue (hunk 1 above)
Comment (carelvwyk) [reply src/components/WalletSelect.vue:40]: I have reached out to VALR to understand whether this would be a concern

**Comment by carelvwyk** [comment]
> The binance QR looks nice but for me it didn't work in all circumstances. I might be wrong but you should probably test all of the following:
> 
> * scan QR with the device camera
> * scan QR with Binance app

TEST RESULTS:
* scan QR with the device camera - gives link to Binance app, launches and launches payment screen
* scan QR with Binance app - launches payment screen

**Review by ckritzinger — APPROVED** [review]
tested Binance, works fine

**Pull request merged**